### PR TITLE
APP-4504 set max width for hint and tooltip

### PR DESCRIPTION
**Pull request by Symphony-cyril**
## Description

Fix max-width 480px for hint and 240px for tooltip

## JIRA ticket

[JIRA_Ticket](https://perzoinc.atlassian.net/browse/APP-4504

## Demo

image

image

### CODE REVIEW

File: src/components/atoms/_tooltip.scss [hunk 1]
```diff
@@ -14,8 +14,9 @@
   padding-top: toRem(2);
   text-align: center;
   width: fit-content;
+  max-width: 240px;
```
Comment (symphony-annaLlorens) [inline src/components/atoms/_tooltip.scss:17]: Same here. Use `toRem(240)`

File: stories/hint.stories.js [hunk 2]
```diff
@@ -3,10 +3,10 @@ export default {
 };
 
 export const Hint = () => {
-  return (`
+  return `
   <div style="min-height: 600px">
     <h2>Hint</h2>
-      <div id="hint-vZUXsOqChv" role="hint" class="tk-hint" style="max-width: 400px; position: relative">
+      <div id="hint-vZUXsOqChv" role="hint" class="tk-hint" style="position: relative">
```
Comment (symphony-annaLlorens) [inline stories/hint.stories.js:9]: Without the relative position is working correctly?

**Comment by Symphony-cyril** [comment]
done

**Review by symphony-annaLlorens — APPROVED** [review]

**Review by symphony-jean-michael — APPROVED** [review]
LGTM

**Pull request merged**